bson-adt-mongo3-async/src/main/scala/adt/bson/mongo/async/client/AsyncClientImplicits.scala

```diff
@@ -0,0 +1,17 @@
+package adt.bson.mongo.async.client
+
+import adt.bson.BsonObject
+import com.mongodb.async.client.MongoDatabase
+
+import scala.language.implicitConversions
+
+trait AsyncClientImplicits {
+
+  implicit def toBsonDatabaseOps(db: MongoDatabase): AsyncBsonDatabaseOps = new AsyncBsonDatabaseOps(db)
+}
+
+class AsyncBsonDatabaseOps(val db: MongoDatabase) extends AnyVal {
+
+  def getBsonCollection(name: String): BsonAdtAsyncCollection =
+    new BsonAdtAsyncCollection(db.getCollection(name, classOf[BsonObject]))
+}
```

Comment:
FAIL: Remove this class. It isn't safe to cast unless the `MongoClient` was created with the `BsonAdtCodecRegistry`